src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetThumbnail/ThumbnailDisplayer/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Image } from 'grommet';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Thumbnail, videoSize } from 'types/tracks';
+import { videoSizeMapping } from 'utils/videoSizeMapping';
+
+interface ThumbnailDisplayerProps {
+  thumbnail: Thumbnail;
+}
+
+const messages = defineMessages({
+  thumbnailAlt: {
+    defaultMessage: 'Live video thumbnail',
+    description:
+      'Accessibility text for the live video thumbnail in the Dashboard.',
+    id: 'components.ThumbnailDisplayer.thumbnailAlt',
+  },
+});
+
+export const ThumbnailDisplayer = ({ thumbnail }: ThumbnailDisplayerProps) => {
+  const intl = useIntl();
+  const urls = thumbnail.urls;
+  const resolutions = Object.keys(urls).map(
+    (size) => Number(size) as videoSize,
+  );
+  return (
+    <Image
+      alt={intl.formatMessage(messages.thumbnailAlt)}
+      fit={'cover'}
+      src={urls[resolutions[0]]}
+      srcSet={resolutions.reduce((acc: string, size: videoSize) => {
+        const url = `${urls[size]} ${videoSizeMapping[size]}w`;
+        if (acc.length === 0) {
+          return url;
+        } else {
+          return `${acc}, ${url}`;
+        }
```

Comment:
```suggestion
        if (acc.length === 0) {
          return `${urls[size]} ${videoSizeMapping[size]}w`;
        }
        return `${acc}, ${url}`;
```

Reply:
Nop, your solution is wrong because `url` is not defined if length is different from `0`

Reply:
of course… :facepalm: